ldap_users/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
2015 for the year. for the authors, please check examples in the OCA 
repository.
thanks!